File: libsignal/service/src/main/java/org/whispersystems/signalservice/internal/push/DeviceInfoList.java
package org.whispersystems.signalservice.internal.push;

import com.fasterxml.jackson.annotation.JsonProperty;

import org.whispersystems.signalservice.api.messages.multidevice.DeviceInfo;

import java.util.List;

public class DeviceInfoList {

  @JsonProperty
  private List<DeviceInfo> devices;

  public DeviceInfoList() {}

  public List<DeviceInfo> getDevices() {
    return devices;
  }
}

File: lib/libsignal-service/src/main/java/org/whispersystems/signalservice/internal/push/DeviceInfoList.kt
/*
 * Copyright 2025 Signal Messenger, LLC
 * SPDX-License-Identifier: AGPL-3.0-only
 */

package org.whispersystems.signalservice.internal.push

import org.whispersystems.signalservice.api.messages.multidevice.DeviceInfo

class DeviceInfoList(
  val devices: List<DeviceInfo> = emptyList()
)
